docs/design/node-resource-reservation-design.md

```diff
@@ -0,0 +1,52 @@
+# Volcano node resource reservation
+## background
+* Consider such situation: there are thounsands of pods to be scheduled every day, in 1 to 2 o'clock 500 low priority pods are created and scheduled which used 99% of cluster resource, in 2 to 3 o'clock 10 high priority pods are created, however, low priority pods are still running, high priority pods can not be scheduled due to lack of resource.
+* Users want high priority tasks in 2 to 3 o'clock have resource to schedule immediately every day and high priority tasks not preempt low priority pods because some low priority pods have already run many days.
+## design
+![annotation](images/node-resource-reservation-annotation.png) 
+### recognize high priority pods
+There are two ways to recognize high priority pods:
+* set annotation volcano.sh/reserveable: true in podgroup 
+which means all pods under podgroup are reserve tasks
+* set annotation volcano.sh/reserveable: true in pod
+which means this pod is reserve task
+### recognize pod max running time
```

Comment:
Is this a maximum execution time limit for a podgroup? If a task exceeds this time, will it be considered a failure?

Reply:
max running time can not only be set in pod but also in podgroup, if in podgroup, it means the max finish time of pods sub the min create time of pods will be less than the max running time. if exceed this time, it will not be considered a failure, the parameter is mainly used to predict whether pod will effect resource reservation later, for example, if the max running time is only 1 minture and we need to reserve 100c after 30 minute, it means it will not effect.